nemo/collections/common/tokenizers/bytelevel_tokenizer.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+from pathlib import Path
+from nemo.collections.common.tokenizers.tokenizer_spec import TokenizerSpec
+
+__all__ = ['ByteLevelTokenizer']
+
+
+class ByteLevelTokenizer(TokenizerSpec):
+    def __init__(self):
+        self.vocab_size = 259
+        self.special_tokens = [self.bos_id, self.eos_id, self.pad_id]
```

Comment:
This seems different from ByT5 design which claim to have [padding, eos, unk]. Also, I am wondering if we should have bert-like cls and msk?

Reply:
i can remove it, but i think it will break tokenization as a bos token is assumed to be present

Reply:
ok, we need bos for decoder in nmt